src/views/OutputView.js

```diff
@@ -0,0 +1,11 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export class OutputView {
+  static printResult(result) {
+    Console.print(result);
+  }
+
+  static printError(ERROR_MESSAGE) {
+    Console.print(ERROR_MESSAGE);
+  }
```

Comment:
```suggestion
  static printMessage(message) {
    Console.print(message);
  }

  static printError(message) {
    Console.print(`[ERROR] ${message}`);
  }
```
개인적으로는 두 메서드가 다른게 없는 것 같아서 조금 더 메서드 이름에 적힌 목적에 맞게 바꿔보는건 어떨까요??